Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -1321,6 +1321,11 @@ public int getMaxUserCacheCount() {
         return config.getBoolean("last-message-reply-recipient", false);
     }
 
+    @Override
+    public boolean isReplyToVanished() {
+        return config.getBoolean("last-message-reply-vanished", true);
```

Comment:
I believe it should be `false` by default
While it is changing behavior that some may not like, I think that it would be better that you won't be able to reply to vanished players (because of the whole vanished concept)